Userland/Libraries/LibGfx/ImageFormats/JPEGXLChannel.h

```diff
@@ -11,14 +11,28 @@
 
 namespace Gfx {
 
+struct ChannelInfo {
+    static ChannelInfo from_size(IntSize size)
+    {
+        return { .width = static_cast<u32>(size.width()),
+            .height = static_cast<u32>(size.height()) };
```

Comment:
clang-format may format this nicer if you include a trailing comma after the 2nd initializer